pkg/apis/migration/v1alpha1/model.go

```diff
@@ -161,3 +167,114 @@ func GetSecret(client k8sclient.Client, ref *kapi.ObjectReference) (*kapi.Secret
 
 	return &object, err
 }
+
+// DeleteLabels - removes migration  labels from the resource after restore
+func DeleteLabels(config *rest.Config, uid string) error {
```

Comment:
The `model.go` is for commonly used convenience functions.  both `DeleteLabels()` and `DeleteMigrated()` seem to fall completely within the concern of, and solely used by, the Migration controller.  What are these defined here?